lib/node_modules/@stdlib/assert/has-same-constructor/README.md

```diff
@@ -0,0 +1,121 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# hasSameConstructor
+
+> Test if two values have same constructor.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var hasSameConstructor = require( '@stdlib/assert/has-same-constructor' );
+```
+
+#### hasSameConstructor( v1, v2 )
+
+Tests if two values have same constructor.
+
+<!-- eslint-disable no-new-wrappers -->
+
+```javascript
+var obj1 = new Date();
+var obj2 = new Date();
+var bool = hasSameConstructor(obj1, obj2);
+// returns true
+
+var obj1 = new Date();
+var obj3 = new String("Hello");
+bool = hasSameConstructor(obj1, obj3);
+// returns false
+
+var obj3 = new String("Hello");
+var obj4 = "World";
+bool = hasSameConstructor(obj3, obj4);
+// returns true
+```
+
+</section>
+
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint-disable no-new-wrappers -->
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var DateObj1 = new Date();
+var DateObj2 = new Date();
+var FloatArrObj = new Float64Array( 10 );
+var IntArrObj = new Int32Array( [ 1, 2, 3 ] );
+var ComplexObj = new Complex64( 5.0, 3.0 );
+
+var bool = hasSameConstructor( DateObj1, DateObj2 );
+// => true
```

Comment:
This is not how we do doctesting. See our doctesting guide: https://github.com/stdlib-js/stdlib/blob/develop/docs/doctest.md